### PR TITLE
Update mqtt.markdown

### PR DESCRIPTION
**Pull request by m4dm4rtig4n**
## Proposed change
Just add my product to use Home Assistant auto-discovery sensor with MQTT.

## Type of change
- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
None

## Checklist
- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by jbouwh** [comment]
I see the page is in French. Is there an English version ?

**Comment by cvalentin-dkt** [comment]
This tool is only for the French community and this one wants the documentation in French :/
I can translate the README.md into English and offer both languages.

**Comment by m4dm4rtig4n** [comment]
This tool is only for the French community and this one wants the documentation in French :/
I can translate the README.md into English and offer both languages.

**Comment by jbouwh** [comment]
> This tool is only for the French community and this one wants the documentation in French :/ I can translate the README.md into English and offer both languages.

That would help, please also explain on your site for what users this is interesting.

**Comment by m4dm4rtig4n** [comment]
I have translate documentation :
https://github.com/MyElectricalData/myelectricaldata#english

File: source/_integrations/mqtt.markdown [hunk 1]
```diff
@@ -548,6 +548,7 @@ The following software has built-in support for MQTT discovery:
 - [HASS.Agent](https://github.com/LAB02-Research/HASS.Agent)
 - [IOTLink](https://iotlink.gitlab.io) (starting with 2.0.0)
 - [MiFlora MQTT Daemon](https://github.com/ThomDietrich/miflora-mqtt-daemon)
+- [MyElectricalData](https://github.com/MyElectricalData/myelectricaldata)
```
Comment (jbouwh) [inline source/_integrations/mqtt.markdown]:
```suggestion
- [MyElectricalData](https://github.com/MyElectricalData/myelectricaldata#english)
```

**Review by jbouwh — CHANGES_REQUESTED** [review]

**Review by jbouwh — APPROVED** [review]
Looks good,
Thnx @m4dm4rtig4n 👍

**Pull request merged**